### PR TITLE
Utopia 1152

### PR DESCRIPTION
**Pull request by antsand**
# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Updating Testing Framework(s)
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

File: src/frontend/src/components/public/PIASideNav/pia-form-sideNav-pages.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ import { buildDynamicPath } from '../../../utils/path';
 import { INavbarItem } from '../../common/Navbar/interfaces';
 import { useLocation } from 'react-router-dom';
 import { useMemo } from 'react';
+import { is } from 'cypress/types/bluebird';
```
Comment (binggaofw) [inline src/frontend/src/components/public/PIASideNav/pia-form-sideNav-pages.ts:14]: you can delete this import, never use this

File: src/frontend/src/components/public/PIASideNav/pia-form-sideNav-pages.ts [hunk 2]
```diff
@@ -45,6 +46,26 @@ export const PiaFormSideNavPages = (
     }
   };
 
+  const enableReview = (): boolean => {
+    if (
+      pia?.hasAddedPiToDataElements === false &&
+      pia?.isNextStepsSeenForDelegatedFlow === true
+    ) {
+      if (isMPORole()) {
```
Comment (binggaofw) [inline src/frontend/src/components/public/PIASideNav/pia-form-sideNav-pages.ts:54]: I thought Brandon said we will get rid of this and start to use roleCheck function, @Bottle7  Can you confirm?

File: src/frontend/src/constant/constant.ts [hunk 3]
```diff
@@ -261,3 +261,15 @@ export enum PiaSorting {
...
+
+interface IapprovalRoles {
+  [key: string]: string;
+}
+
+export const ApprovalRoles: IapprovalRoles = {
+  PROGRAM_MANAGER: 'Program Manager / Product Owner',
+  DIRECTOR: 'Director',
+  EX_DIRECTOR: 'Executive Director',
+  MOIS: 'Ministry Chief Information Officer',
```
Comment (binggaofw) [inline src/frontend/src/constant/constant.ts:273]: MCIO?  MOIS more like a typo but I could be wrong

**Comment by binggaofw** [comment]
the review folder under PIAFormTabs does not commit.

File: src/frontend/src/components/public/PIAFormTabs/review/index.tsx [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+
+  const initialFormState: IReview = useMemo(
+    () => ({
+      programArea: {
+        selectedRoles: [],
+      },
+      mpo: {
+        isAcknowledged: false,
+        reviewNote: '',
+        ReviewedBy: '',
```
Comment (Bottle7) [inline src/frontend/src/components/public/PIAFormTabs/review/index.tsx:26]: Can we keep names camel cased for congruency?

File: src/frontend/src/components/public/PIASideNav/pia-form-sideNav-pages.ts (hunk 2 above)
Comment (Bottle7) [reply src/frontend/src/components/public/PIASideNav/pia-form-sideNav-pages.ts:53]: That was the idea, yeah.

File: src/frontend/src/components/public/PIAFormTabs/review/index.tsx [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+  };
+
+  // passing updated data to parent for auto-save to work efficiently only if there are changes
+  useEffect(() => {
+    const getPIADATA = async () => {
+      const piaData = await HttpRequest.get(
+        API_ROUTES.GET_PIA_INTAKE.replace(':id', `${id}`),
+      );
+      const data: IPiaForm = Object(piaData).data;
+      console.log(data);
```
Comment (Bottle7) [inline src/frontend/src/components/public/PIAFormTabs/review/index.tsx:54]:
```suggestion

```

File: src/frontend/src/components/public/PIAFormTabs/review/interfaces.ts [hunk 6]
```diff
@@ -0,0 +1,12 @@
+export interface IReview {
+  programArea: {
+    selectedRoles: string[];
+  };
+  mpo: {
+    isAcknowledged: boolean;
+    reviewNote: string;
+    ReviewedBy?: string;
```
Comment (Bottle7) [inline src/frontend/src/components/public/PIAFormTabs/review/interfaces.ts:8]: Please change these props to camel case for consistency.

**Review by binggaofw — APPROVED** [review]

**Pull request merged**